### PR TITLE
Initial CI enhancements

### PR DESCRIPTION
**Pull request by sabre1041**
Initial CI enhancements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,31 +57,27 @@ After you've entered the auto code, the credentials are stored in *gdrive_temp/.
 
 
 
-### Deploying the application
+## Deploying the application
```
Comment (etsauer) [inline README.md]: @sabre1041 shouldn't this section be part of `## Deploying to OpenShift`?

File: .openshift/templates/jenkins-ephemeral.yml [hunk 2]
```diff
@@ -0,0 +1,241 @@
+apiVersion: template.openshift.io/v1
```
Comment (etsauer) [inline .openshift/templates/jenkins-ephemeral.yml]: why carry this vs use the one embedded in the cluster?

File: README.md [hunk 3]
```diff
@@ -101,12 +93,12 @@ Utilize the following steps to deploy the project
...
     ``` 
  
-3. Update credentials in the _applier/params/community-ninja-board-deployment_ file
+3. Ensure the variables have been updated in the _ninja-board-deployment_ params file
 
 4. Execute the _openshift-applier_
 
     ```
-    ansible-playbook -i applier/inventory galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml
+    ansible-playbook -i applier/inventory galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e="@.openshift/params/ninja-board-deployment
```
Comment (etsauer) [inline README.md]: @sabre1041 this command doesn't make sense.

Comment (etsauer) [reply README.md]: should be `.applier/inventory`

File: .openshift/templates/jenkins-ephemeral.yml (hunk 2 above)
Comment (sabre1041) [reply .openshift/templates/jenkins-ephemeral.yml]: We are mounting the global library xml ConfigMap

File: README.md (hunk 1 above)
Comment (sabre1041) [reply README.md]: Sure. Will update based on other feedback

File: README.md (hunk 3 above)
Comment (sabre1041) [reply README.md]: done###

**Comment by sabre1041** [comment]
@etsauer updated based on your feedback

File: README.md [hunk 4]
```diff
@@ -101,12 +92,12 @@ Utilize the following steps to deploy the project
...
     ``` 
  
-3. Update credentials in the _applier/params/community-ninja-board-deployment_ file
+3. Ensure the variables have been updated in the _ninja-board-deployment_ params file
 
 4. Execute the _openshift-applier_
 
     ```
-    ansible-playbook -i applier/inventory galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml
+    ansible-playbook -i .applier/inventory galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e="@.openshift/params/ninja-board-deployment
```
Comment (etsauer) [inline README.md]: need double quotes at the end of this line.

Comment (sabre1041) [reply README.md]: done

**Pull request merged**